packages/client/src/products/types/facetValue.types.ts

```diff
@@ -6,6 +6,7 @@ export type FacetValue = {
   url: string;
   parentId: number;
   groupsOn: number;
+  groupType?: number;
```

Comment:
Is this correct? I think this is only being added to the FacetEntity by the facets schema by the fetchProductListingFacetsFactory module.

Reply:
Indeed, this parameter addition wasn't needed, thanks, fixed!